src/ol_social_auth/BUILD

```diff
@@ -0,0 +1,21 @@
+python_sources(
+    name="ol_social_auth_source",
+    dependencies=["//:external_dependencies#social-auth-core"]
```

Comment:
pants was not detecting `social-auth-core` automatically, therefore I added it here.

Reply:
yes